### PR TITLE
[FIX] mass_mailing: handle comma-separated template styles 

### PR DESCRIPTION
**Pull request by Zynton**
To give a custom style to a template, we need to add a style element at its top, with the id "design-element", and apply to it the supported styles. These supported styles and their selectors can be seen in the `DEFAULT_CSS_OBJECT` constant in `mass_mailing_snippets.js`. However, comma-separated selectors were not supported, meaning styles could only be incomplete.

Inline styles applied by the editor should always have priority over the general stylesheet. This failed in the case of nested styles (a <font> with a color within a <span> with a font size for instance).

task-2816655

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mass_mailing/views/snippets_themes.xml [hunk 1]
```diff
@@ -629,28 +629,28 @@
         <we-row string="Text" class="o_design_tab_title">
             <we-input data-customize-css-property=""
                         data-css-property="font-size"
-                        data-selector-text="p, p *, li, li *"
+                        data-selector-text="p, p > *, li, li > *"
```
Comment (dmo-odoo) [inline addons/mass_mailing/views/snippets_themes.xml:632]: Même question ici. A quel point c'est pété ou pas pété avec le nouveau js mais l'ancien xml ?

File: addons/mass_mailing/views/snippets_themes.xml [hunk 2]
```diff
@@ -669,30 +669,30 @@
         <we-row string="Primary Buttons" class="o_design_tab_title">
             <we-input data-customize-css-property=""
                         data-css-property="font-size"
-                        data-selector-text="a[href].btn-primary, a[href].btn-outline-primary, a[href].btn-fill-primary"
+                        data-selector-text="a.btn-primary, a.btn-outline-primary, a.btn-fill-primary"
```
[the remark this answers is not in the document]
Comment (Zinston) [reply addons/mass_mailing/views/snippets_themes.xml:672]: Ca pètera la reconnaissance des styles dans l'onglet Design. Mais on est en master ici donc c'est pas un problème vu que la feature a été ajoutée dans la même version.

[the remark this answers is not in the document]
Comment (dmo-odoo) [reply addons/mass_mailing/views/snippets_themes.xml:672]: Ah my bad, j'ai tellement l'habitude que nos branches soient en 15... ^^'

**Comment by dmo-odoo** [comment]
@robodoo rebase-ff r+

**Comment by dmo-odoo** [comment]
@robodoo r-

**Comment by dmo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**